rundeckapp/grails-app/i18n/messages.properties

```diff
@@ -1237,6 +1237,7 @@ set.as.default.filter=Set as Default Filter
 remove.default.filter=Remove Default Filter
 run.a.command.on.count.nodes.ellipsis=Run a command on {0} {1} …
 not.authorized=Not authorized
+user.not.allowed=User not allowed
```

Comment:
Hmm , just a suggestion for rephrasing, but not necessary.
```suggestion
user.not.allowed=Authorization was not within the required role.
```

Reply:
I would be concerned that this might be giving a user too much information.